wrapanapi/scvmm.py

```diff
@@ -225,6 +226,17 @@ def vm_creation_time(self, vm_name):
     def info(self, vm_name):
         pass
 
+    def vm_hardware_configuration(self, vm_name):
+        data = self.run_script(
+            """
+            $vm = Get-SCVirtualMachine -Name \"{}\"
+            $conf = @{{"ram"=$vm.Memory; "cpu"=$vm.CPUCount}}
+            $conf|ConvertTo-Json -Depth 3 -Compress
+            """.format(vm_name))
+        result = json.loads(data)
+        return {str(key): (str(val) if isinstance(val, unicode) else val) for key, val in
```

Comment:
.loads return unicode strings - reformatting them for better output in reports

Reply:
Actually it would probably be more correct to use `.encode('utf-8')` on unicode values, but I guess it is an equivalent operation here.